task/buildah-remote/0.1/buildah-remote.yaml

```diff
@@ -59,6 +59,13 @@ spec:
     description: The image is built from this commit.
     name: COMMIT_SHA
     type: string
+  - default: repos.d
+    description: Path in the repository in which yum repodir files are stored
+    name: REPOSDIR_SRC
```

Comment:
Can the description be more specific instead of using abbrev like `repodir`?

My understanding is this is a directory containing custom YUM repository configuration files.

Should/Must this directory be in a relative path? At least, from the update to the script below, looks it is. 

How is the default value `repos.d` determined? Could a different directory name break something? Any other service depends on it?

Could you please update the description to give more information? Update both tasks.

In addition, how is the content of this directory generated and by what service?

Reply:
> Can the description be more specific instead of using abbrev like repodir?

I'll update the description.

> Should/Must this directory be in a relative path?

Having it as an absolute path means that the user needs to understand more about the workspace definitions inside the task and their pipelinerun. I think it also means more limited portability. With that in mind, I think relative path makes sense. Let me know if you see it differently.

> How is the default value repos.d determined? Could a different directory name break something? Any other service depends on it?

It can be something else.

> In addition, how is the content of this directory generated and by what service?

The service that generates such configs within Red Hat is ODCS, but there could be other services to generate it, and it can even be created manually. We're finalizing a task to have that automated and integrated inside the build pipeline. It's implemented on a different repo, as we need access to the internal network to test it. We're also working on a process for automatically mirroring changes made to that task to build-definitions, so it can be consumed like any other task.

This is the task: https://github.com/redhat-appstudio/tekton-tools/blob/main/tasks/generate-odcs-compose.yaml